src/PageLayout/Navigation/NavBarItem.js

```diff
@@ -3,29 +3,28 @@ import React, {Component} from 'react';
 import './NavBarItem.css';
 
 class NavBarItem extends Component {
-    constructor(){
-    super();
-
-    this.state = {
-        highlighted: false
-      }
-    }
-
-    changeColor(){
-        this.setState({highlighted: !this.state.highlighted})
-    }
-
     render(){
-        let lineColor = this.state.highlighted ? '#5DC0F9' : ''
-        let itemColor = this.state.highlighted ? '#F2F2F2' : ''
+        let lineColor = this.props.highlighted ? '#5DC0F9' : ''
+        let itemColor = this.props.highlighted ? '#F2F2F2' : ''
         return(
-            <div className="navbar-item" style={{background: itemColor}} onClick={this.changeColor.bind(this)}>
-            {/* <div className="navbar-item" style={{background: itemColor}}> */}
-                <div style={{background: lineColor}} className="left-line"></div>
-                <img src={this.props.icon} className="navbar-icon" alt="icon"/>
-                <p className="navbar-name">{this.props.item}</p>
+            <div>
+                {this.props.item === "signout" &&
+                    <div className="navbar-item signout" onClick={this.props.handler}>
+                        <img src={this.props.icon} className="navbar-icon" alt="icon"/>
+                        <p className="navbar-name">Sign Out</p>
+                    </div>
+                }
+                {this.props.item !== "signout" &&
+                    <div className="navbar-item" style={{background: itemColor}} onClick={() => this.props.handler(this.props.item)}>
+                        <div style={{background: lineColor}} className="left-line"></div>
+                        <img src={this.props.icon} className="navbar-icon" alt="icon"/>
+                        <p className="navbar-name">{this.props.item}</p>
+                    </div>
+                }
```

Comment:
instead of two condition checks you should be able to do:
```
{this.props.item === "signout" ?
    <div className="navbar-item signout" ...
    ...
    </div>
:
    <div className="navbar-item" ...
    ...
    </div>
}